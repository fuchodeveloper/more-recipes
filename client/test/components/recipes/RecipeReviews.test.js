import React from 'react';
import { shallow } from 'enzyme';
import expect from 'expect';
import { RecipeReviews }
  from '../../../components/recipes/RecipeReviews';

let props;

const setup = () => {
  props = {
    review: {
      review: ''
    }
  };

  return shallow(<RecipeReviews {...props} />);
};

describe('RecipeReviews component', () => {
  it('should render the right amount of elements', () => {
    const wrapper = setup();
    expect(wrapper).toMatchSnapshot();
  });
});
