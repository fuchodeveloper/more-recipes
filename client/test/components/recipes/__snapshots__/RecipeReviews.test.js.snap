// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RecipeReviews component should render the right amount of elements 1`] = `
<div>
  no reviews
</div>
`;
